snap/container_test.go

```diff
@@ -45,13 +46,13 @@ func (s *FileSuite) TestFileOpenForSnapDir(c *C) {
 	err = ioutil.WriteFile(snapYaml, []byte(`name: foo`), 0644)
```

Comment:
these tests maybe to should move to snapfile?

Reply:
mmm I have some tests in snapfile_test.go already that duplicate TestFileOpenForSnapDir, but I guess I don't have the equivalent of TestFileOpenForSnapDirErrors, I could move that over.